src/test/java/uk/gov/justice/hmpps/prison/api/resource/impl/OffenderAssessmentResourceIntTest.kt

```diff
@@ -0,0 +1,813 @@
+package uk.gov.justice.hmpps.prison.api.resource.impl
```

Comment:
converted from java and added nested tests - additional tests to be added separately